### DISCUSSION TITLE
Mapset score system

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#18360] [Ideas]
The lazer has a "mapset panel" system, which is displayed above all the difficulties (where the cursor is located in the screenshot).

Some games have a system where an "average" score is displayed at this location, which is usually equal to the lowest score of all achieved in this section (yeah, that doesn't quite sound like "average").

In the case of osu, this may reflect the rank achieved on all difficulties. If you have reached the "C" rank everywhere, then the "C" rank will be displayed on the general panel. If the example is as in the screenshot (SS, several S and B), then B rank will be displayed. 
![image](https://user-images.githubusercontent.com/59089206/169689369-aa96f886-1d3d-4d9d-b699-898a5ee70837.png)
Additionally, this gives a little more information to the panel itself.

This function can also be relevant for a site where there is already a filter by achieved rank. So this could be added for the mapset as well.
![image](https://user-images.githubusercontent.com/59089206/169689291-3ab1f669-27b1-4077-989d-ebf139791d28.png)

P.S. Can also calculate the average rank, and not take the lowest one, but for this you need to come up with a counting system.

**Comment by peppy** [comment]
It's not the right time to be adding even more new systems to the game, so if this was to be considered it would be a far-future consideration.
